src/core/congestion_control.h

```diff
@@ -12,9 +12,9 @@ typedef struct QUIC_ACK_EVENT {
 
     uint64_t TimeNow; // microsecond
 
-    uint64_t LargestPacketNumberAcked;
+    uint64_t LargestAck;
```

Comment:
LargestAckedPacketNumber?

Reply:
Yeah this discrepancy occurred to me too, but we have several other names in the loss detection code that are derived from LargestAck (such as LargestAckEncryptLevel, NewLargestAckRetransmittable, NewLargestAckDifferentPath), which is why I opted here for using the original name LargestAck. Either way is kinda bad...